src/components/CartProductCard.js

```diff
@@ -0,0 +1,33 @@
+import "./styles/CartProductCard.css"
+
+const CartProductCard = (props) => {
+    return (
+        <div className="cardHorizotalContainer">
+            <div className="cardHorizontal">
+                <div className="cardWithBadge cardImgContainer">
+                    <img className="cardImgCart" src={props.img} alt="Product Img" />
+                </div>
+                <div>
+                    <p class="cardDetails">
+                        <span className="cardName">{props.name}</span>
+                        <br />
```

Comment:
https://stackoverflow.com/a/1726103

Reply:
I'll change it and use different elements for that.